packages/db/src/index.ts

```diff
@@ -10,6 +10,9 @@ export const schema = { ...auth, ...post };
 export { mySqlTable as tableCreator } from "./schema/_table";
 
 export * from "drizzle-orm/expressions";
+export * from "drizzle-orm/sql/functions";
+export { sql } from "drizzle-orm/sql/sql";
```

Comment:
What "extras" are we getting if we just export the entire `/sql` dir?

Reply:
Hello everyone :)

the type SQL could also be added 
export type { SQL } from "drizzle-orm/sql/sql"

Reply:
prob best to just `export * from "drizzle-orm/sql"`

Reply:
> What "extras" are we getting if we just export the entire `/sql` dir?

All of these:

![Screenshot 2024-04-10 150700](https://github.com/t3-oss/create-t3-turbo/assets/72540492/49130b44-4150-4c00-8b96-a55396d356fe)
![Screenshot 2024-04-10 150713](https://github.com/t3-oss/create-t3-turbo/assets/72540492/29088723-1a0b-4f3b-92b2-f8cb342e2db6)

None of them seem necessary.